src/Datasource/EntityTrait.php

```diff
@@ -1231,7 +1231,7 @@ public function source($alias = null)
      */
     public function __toString()
     {
-        return json_encode($this, JSON_PRETTY_PRINT);
+        return json_encode($this, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
```

Comment:
That does take into account scenarios where people are sprintf() or concatting entities into log messages though.